src/controllers/albumController.js

```diff
@@ -4,17 +4,48 @@ const CustomError = require('../errors');
 const logger = require('../../logs/logger');
 
 const createAlbum = async (req, res) => {
+  /*
+     #swagger.summary = 'Create new album and save it to the database'
+    
+     #swagger.responses[201] = {
+				description: 'Albums was successfully created.',
+        schema: [{ $ref: '#/definitions/Album'' }]
```

Comment:
According to the return value of this function, we return an object with key "album" and value being the album that was just created: `res.status(StatusCodes.CREATED).json({ album });`

So this should not be an array here:

```suggestion
        schema: { album: { $ref: '#/definitions/Album' } }
```

Reply:
Also note that you had an extra quotation mark after definitions/Album that was causing swagger-autogen to not be able to parse somethings correctly here